docs/content/latest/benchmark/sysbench-ysql.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Benchmark YSQL performance using Sysbench
+headerTitle: Sysbench
+linkTitle: Sysbench
+description: Benchmark YugabyteDB YSQL performance using Sysbench.
+headcontent: Benchmark YugabyteDB YSQL performance using Sysbench.
+menu:
+  latest:
+    identifier: sysbench-ysql
+    parent: benchmark
+    weight: 5
+aliases:
+  - /benchmark/sysbench/
+showAsideToc: true
+isTocNested: true
+---
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="/latest/benchmark/tpcc-ysql/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL
+    </a>
+  </li>
+
+</ul>
+
+Sysbench is a popular tool for benchmarking system capabilities like CPU, memory and I/O as well as databases like Postgres and MySQL.
+Follow the steps below to run sysbench against YugabyteDB.
+The version we are using is forked from the [official](https://github.com/akopytov/sysbench) version with a few additions to better reflect YugabyteDB's distributed nature.
+
+## Step 1. Get Sysbench.
+
+You can do this by running the following commands.
+
+```sh
+$ cd $HOME
+$ git clone https://github.com/yugabyte/sysbench.git
+$ cd sysbench
+$ ./autogen.sh && ./configure --with-pgsql && make -j && sudo make install
```

Comment:
Not needed for this PR but we should make `--with-pgsql` the default in our fork.

Reply:
Sure.